### PR TITLE
APDX-10469 Actualize SDK swift example

### PR DESCRIPTION
**Pull request by dimasno1**

### CODE REVIEW

File: BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/AdLoadController.swift [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import UIKit
+
+extension AdLoadController {
+    enum State {
+        case idle, loading, loaded
+    }
+}
+
+class AdLoadController: UIViewController {
+    var topTitle: String? {
```
Comment (staskochkin) [inline BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/AdLoadController.swift:14]: Do we need a computed property that returns nil?

File: BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/InterstitialViewController.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    private func deleteLoadedAd() {
+        interstitial = nil
+    }
+}
+
+extension InterstitialViewController: GADFullScreenContentDelegate {
+    func adDidRecordImpression(_ ad: any GADFullScreenPresentingAd) {
+
```
Comment (staskochkin) [inline BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/InterstitialViewController.swift:73]: Please add logs to delegate methods

File: BidMachineSampleSwift/BidMachineSampleSwift/ApplicationSupport/Info.plist [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>GADApplicationIdentifier</key>
+    <string>ca-app-pub-3216013768320747~6578253232</string>
```
Comment (staskochkin) [inline BidMachineSampleSwift/BidMachineSampleSwift/ApplicationSupport/Info.plist:6]: Don't leave the `GADApplicationIdentifier` in the public repo.

File: BidMachineSampleSwift/BidMachineSampleSwift/Constant.swift [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import Foundation
+
+enum Constant {
+    enum UnitID {
+    }
+}
+
+extension Constant.UnitID {
+    enum Banner {
+        static let googleTest = "ca-app-pub-3940256099942544/2435281174"
```
Comment (staskochkin) [inline BidMachineSampleSwift/BidMachineSampleSwift/Constant.swift:17]: Don't leave ad unit IDs

Comment (dimasno1) [reply BidMachineSampleSwift/BidMachineSampleSwift/Constant.swift:17]: These are ad unit IDs provided by Google for testing purposes. They are publicly available [here](https://developers.google.com/admob/ios/test-ads)

File: BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/AdLoadController.swift (hunk 1 above)
Comment (dimasno1) [reply BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/AdLoadController.swift:14]: It's base class. Subclasses override it.

File: BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/InterstitialViewController.swift (hunk 2 above)
Comment (dimasno1) [reply BidMachineSampleSwift/BidMachineSampleSwift/AdControllers/InterstitialViewController.swift]: ✅

File: BidMachineSampleSwift/BidMachineSampleSwift/ApplicationSupport/Info.plist (hunk 3 above)
Comment (dimasno1) [reply BidMachineSampleSwift/BidMachineSampleSwift/ApplicationSupport/Info.plist]: ✅

**Review by staskochkin — APPROVED** [review]

**Pull request merged**